be/PodoCareWebsite/src/main/java/com/podocare/PodoCareWebsite/controller/order/BrandController.java

```diff
@@ -0,0 +1,71 @@
+package com.podocare.PodoCareWebsite.controller.order;
+
+import com.podocare.PodoCareWebsite.model.Brand_Supplier.Brand;
+import com.podocare.PodoCareWebsite.model.Brand_Supplier.DTOs.BrandDTO;
+import com.podocare.PodoCareWebsite.model.Brand_Supplier.DTOs.BrandFilterDTO;
+import com.podocare.PodoCareWebsite.repo.order.BrandRepo;
+import com.podocare.PodoCareWebsite.service.order.BrandService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/brands")
+public class BrandController {
+    @Autowired
+    BrandService brandService;

```

Comment:
don't use @Autowired, change to 
```suggestion
@Controller
@RequiredArgsConstructor
@RequestMapping("/brands")
public class BrandController {

    private final BrandService brandService;

```
change it in every place